Shared/Realm.Shared/exceptions/RealmInvalidTransactionException.cs

```diff
@@ -19,7 +19,8 @@
 namespace Realms
 {
     /// <summary>
-    /// An exception, thrown when trying to write data to the Realm and you haven't begun a Write transaction or when the realm is opened as read-only.
+    /// An exception, thrown when trying to write data to the <see cref="Realm"/> and you haven't begun a Write 
+    /// <see cref="Transaction"/> or when the <see cref="Realm"/> is opened as read-only.
```

Comment:
Is this the exception that is thrown when trying to create a transaction on a readonly realm? Because if not, I think the case is covered by the comment above. Also, it could be reworded to "when trying to write data the realm outside a transaction" or similar.